src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -320,6 +322,17 @@ public String getInvalidityInfoForEmail(String email) {
         return "";
     }
 
+    /**
+     * Checks if {@code gracePeriod} is not negative.
+     * @return An explanation why the {@code gracePeriod} is not acceptable.
+     */
```

Comment:
For consistency with the other methods, add a line `Returns an empty string if the {@code gracePeriod} is acceptable.`